### PR TITLE
Find Place sample

### PR DESCRIPTION
**Pull request by nCastle1**
Implementation of the Find Place sample across all platforms.

### CODE REVIEW

File: src/Forms/Shared/Samples/Search/FindPlace/FindPlace.xaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="2*" />
+        </Grid.ColumnDefinitions>
+        <Label Text="Search" Grid.Row="0" Grid.Column="0" />
+        <Label Text="Location" Grid.Row="1" Grid.Column="0" />
+        <SearchBar x:Name="MySearchBox" IsEnabled="False"  Grid.Row="0" Grid.Column="1" Grid.ColumnSpan="2" TextChanged="MySearchBox_TextChanged" Unfocused="MyLocationBox_Unfocused" Focused="MySearchBox_Focused" />
```
Comment (ThadT) [inline src/Forms/Shared/Samples/Search/FindPlace/FindPlace.xaml]: Maybe more descriptive control names (?). MySearchBox => SearchTextBox, MyLocationBox => SearchLocationBox, etc.

File: src/Forms/Shared/Samples/Search/FindPlace/FindPlace.xaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <SearchBar x:Name="MyLocationBox" Text="Current Location" IsEnabled="False"   Grid.Row="1" Grid.Column="1" Grid.ColumnSpan="2" TextChanged="MyLocationBox_TextChanged" Unfocused="MyLocationBox_Unfocused" Focused="MySearchBox_Focused" />
```
Comment (ThadT) [inline src/Forms/Shared/Samples/Search/FindPlace/FindPlace.xaml]: Better formatting of XAML elements (?). For example:
```` XAML    
<Button x:Name="MySearchButton" 
              Grid.Column="0" Grid.ColumnSpan="2" Grid.Row="2"
              Text="Search All"
              IsEnabled="False"
              Clicked="MySearchButton_Clicked" />    
````

File: src/Forms/Shared/Samples/Search/FindPlace/FindPlace.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+
+            // Define the callout
+            CalloutDefinition calloutBody = new CalloutDefinition(title, address);
+
+            // Show the callout on the map at the tapped location
+            MyMapView.ShowCalloutAt(e.Location, calloutBody);
+        }
+
+        /// <summary>
+        /// Returns a list of suggestions based on the input search text and limited by the specified paramters
```
Comment (ThadT) [inline src/Forms/Shared/Samples/Search/FindPlace/FindPlace.xaml.cs]: Typo, "paramters"

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/Search/FindPlace/FindPlace.xaml.cs [hunk 4]
```diff
@@ -0,0 +1,355 @@
...
+
+            // Define the callout
+            CalloutDefinition calloutBody = new CalloutDefinition(title, address);
+
+            // Show the callout on the map at the tapped location
+            MyMapView.ShowCalloutAt(e.Location, calloutBody);
+        }
+
+        /// <summary>
+        /// Returns a list of suggestions based on the input search text and limited by the specified paramters
```
Comment (ThadT) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/Search/FindPlace/FindPlace.xaml.cs]: Typo, "paramters"

File: src/iOS/Xamarin.iOS/Samples/Search/FindPlace/FindPlace.cs [hunk 5]
```diff
@@ -0,0 +1,597 @@
...
+
+            // Define the callout
+            CalloutDefinition calloutBody = new CalloutDefinition(title, address);
+
+            // Show the callout on the map at the tapped location
+            _myMapView.ShowCalloutAt(e.Location, calloutBody);
+        }
+
+        /// <summary>
+        /// Returns a list of suggestions based on the input search text and limited by the specified paramters
```
Comment (ThadT) [inline src/iOS/Xamarin.iOS/Samples/Search/FindPlace/FindPlace.cs]: Typo, "paramters"

**Comment by nCastle1** [comment]
I'm closing this issue as the code has been merged into master with all requested changes. Thanks @ThadT ! :)

**Pull request merged**